src/app/components/cells/link/getVisibleLinkCount.js

```diff
@@ -1,12 +1,14 @@
 // TODO: Keep in sync with link label spacings/font
-const gapWidthInPx = 8;
+const gapWidthInPx = 6;
 const linkLabelStyle = {
   padding: "6px 8px",
   fontFammily: "Roboto",
   fontSize: "1.3rem",
   fontWeight: "normal",
   position: "absolute"
 };
+const cellPaddingInPx = 10;
```

Comment:
Könnte man die beiden Style-Werte evtl. noch mit dem CSS verknüpfen? Wenn das nur an 1-2 Stellen vorkommt wäre das sicher nicht schlecht; wenn man dafür allerdings Unmengen CSS-Files anfassen muss lohnt es sich wohl kaum.

Reply:
Was meinst du genau mit `mit dem CSS verknüpfen`?

Reply:
Wie besprochen ist das out of scope :)